src/components/ZclAttributeManager.vue

```diff
@@ -410,97 +417,115 @@ export default {
         rowsPerPage: 0,
         sortBy: 'clientServer',
       },
-      columns: [
-        {
-          name: 'status',
-          required: false,
-          label: '',
-          align: 'left',
-          style: 'width:1%',
-        },
-        {
-          name: 'included',
-          label: 'Enabled',
-          field: 'included',
-          align: 'left',
-        },
-        {
-          name: 'attrID',
-          align: 'left',
-          label: 'Attribute ID',
-          field: 'attrID',
-          sortable: true,
-          style: 'max-width: 90px',
-          headerStyle: 'max-width: 90px',
-        },
-        {
-          name: 'attrName',
-          label: 'Attribute',
-          field: 'attrName',
-          align: 'left',
-          sortable: true,
-        },
-        {
-          name: 'required',
-          label: 'Required',
-          field: 'required',
-          align: 'left',
-          sortable: true,
-        },
-        {
-          name: 'clientServer',
-          label: 'Client/Server',
-          field: 'clientServer',
-          align: 'left',
-          sortable: true,
-        },
-        {
-          name: 'mfgID',
-          label: 'Mfg Code',
-          align: 'left',
-          field: 'mfgID',
-          sortable: true,
-        },
-        {
-          name: 'storageOption',
-          label: 'Storage Option',
-          align: 'left',
-          field: 'storageOption',
-          sortable: true,
-        },
-        {
-          name: 'singleton',
-          align: 'left',
-          label: 'Singleton',
-          field: 'singleton',
-          sortable: true,
-        },
-        {
-          name: 'bounded',
-          align: 'left',
-          label: 'Bounded',
-          field: 'bounded',
-          sortable: true,
-        },
-        {
-          name: 'type',
-          align: 'left',
-          label: 'Type',
-          field: 'type',
-          sortable: true,
-        },
-        {
-          name: 'default',
-          align: 'left',
-          label: 'Default',
-          field: 'default',
-          style: 'min-width: 180px',
-          headerStyle: 'min-width: 180px',
-        },
-      ],
+      columns: [],
       forcedExternal: [],
+      enableSingleton: false,
+      enableBounded: false,
     }
   },
+  mounted() {
+    const enableZigbeeFeatures =
```

Comment:
I always complain if UI is polluted by words "zigbee" and "matter". UI should be driven by "features" and which of those are "zigbee" and which are "matter" is defined in the `zcl.json.

Reply:
So the way this should be implemented is not like this...

You should do the following:
   1.) Matter UI options should say: "singletonsEnabled=false" inside matter zcl.json file.
   2.) ZIgbee UI options should say: "singletonsEnabled=true" inside zigbee zcl.json file.
   3.) UI should then simply act on this data, not have this hardcoded business logic of:
      if category is matter, then disable singletons.

Reply:
the logic is the same either way